packages/tables/docs/02-columns/08-text-input.md

```diff
@@ -49,3 +49,14 @@ TextInputColumn::make()
         // Runs after the state is saved to the database.
     })
 ```
+
+## Prefix and suffix support
+
+The text input column supports prefix and suffix elements:
```

Comment:
Please expand the docs to copy the text we already have on the forms/text-input page, which fully explains the features